src/inference/src/os/win/win_system_conf.cpp

```diff
@@ -167,15 +172,15 @@ void parse_processor_info_win(const char* base_ptr,
                 if (_processors <= list[list_len - 1] + base_proc) {
                     group_start = list[0];
                     group_end = list[list_len - 1];
-                    group_id = group;
+                    group_id = CPU_BLOCKED;
                     group_type = EFFICIENT_CORE_PROC;
                 }
                 for (int m = 0; m < _processors - list[0]; m++) {
                     _cpu_mapping_table[list[m] + base_proc][CPU_MAP_CORE_TYPE] = EFFICIENT_CORE_PROC;
-                    _cpu_mapping_table[list[m] + base_proc][CPU_MAP_GROUP_ID] = group;
-                    _proc_type_table[0][EFFICIENT_CORE_PROC]++;
+                    _cpu_mapping_table[list[m] + base_proc][CPU_MAP_GROUP_ID] = group_id;
+                    _cpu_mapping_table[list[m] + base_proc][CPU_MAP_USED_FLAG] = CPU_BLOCKED;
                 }
-                group++;
+                num_blocked++;
```

Comment:
num_blocked value is for ecore number or ecore group number?

Reply:
num_blocked value is for blocked ecore number